index.d.ts

```diff
@@ -354,7 +354,15 @@ export declare interface TracerOptions {
   flushInterval?: number;
 
   /**
-   *  Number of spans before partially exporting a trace. This prevents keeping all the spans in memory for very large traces.
+   * Enables or disables large payloads being sent to APM.
```

Comment:
not sure about wording here, so any suggestions would be appreciated! Whatever is finalized for wording here is what I'll use for the documentation repo